### DISCUSSION TITLE
Bracket pair completion?

### DISCUSSION THREAD

**Discussion by cloin** [ansible/ansible-navigator#590] [Ideas]
Would it be worth it to implement bracket pairing on the input line? For example: if I am viewing docs on a module and I want to jump to displaying just examples I can type out `{{ examples }}`. I'm not sure how popular it is to interact with navigator in this way, but if there's a push to use it this way, it would a nice to have feature.

**Comment by cidrblock** [comment]
@cloin If I understand this, once you type `{{` you are suggesting navigator add a `}}` and leave the cursor inside it ( `{{ | }}` )

I think it's a cool idea and would be tricky/fun to code :)  I'll add an issue so it's not forgotten

**Reply by webknjaz** [reply]
Ref https://github.com/ansible/ansible-navigator/issues/591
